01-CORE/07-INTEGRATIONS/bridges/secure_proxy/core/vault_manager.py

```
@@ -0,0 +1,549 @@
+        
+        # Cloud provider credentials
+        self.azure_credential = None
+        self.aws_session = None
+        self.gcp_credentials = None
+        
+        # Cached credentials
+        self.credential_cache: Dict[str, CloudCredentials] = {}
+        self.cache_ttl = 300  # 5 minutes
+        
+        # Initialize Vault connection
+        asyncio.create_task(self._initialize_vault())
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)